test/filters/browser_tests.filter

```diff
@@ -112,6 +115,27 @@
 #
 # The binder is not instantiated because BraveContentBrowserClient is also not
 # available.
+-All/MLPromotionInstallDialogBrowserTest.MlHandlesInvisible/Detailed_Install_Dialog
```

Comment:
this shouldn't be happening anymore, what is the exact error here?

Reply:
that should only happen if the test creates its own ContentBrowserClient subclass and I doubt that is happening for all of these because it was pretty rare before 120. If we're seeing this in other tests than something else changed that we need to fix because this issue is effectively resolved for most upstream tests now.

Reply:
Removed most tests from the filter in https://github.com/brave/brave-core/pull/20434/commits/bfade20804c52f7f7ec4cc9de9960105e4ed6434 and added a `chromium_src` override for `chrome/browser/storage/shared_storage_browsertest.cc`